metrics/updater.go

```diff
@@ -3,47 +3,53 @@ package metrics
 func IncrementBinlogBackupCountMetrics(clusterName string) {
 	binlogBackupCountMetrics.WithLabelValues(clusterName).Inc()
 }
-
 func IncrementBinlogBackupFailureCountMetrics(clusterName string, action string) {
 	binlogBackupFailureCountMetrics.WithLabelValues(clusterName, action).Inc()
 }
-
 func UpdateBinlogBackupDurationSecondsMetrics(clusterName string, durationSeconds float64) {
 	binlogBackupDurationSecondsMetrics.WithLabelValues(clusterName).Observe(durationSeconds)
 }
+func SetBinlogBackupInProgressMetrics(clusterName string, inProgress bool) {
+	if inProgress {
+		binlogBackupInProgressMetrics.WithLabelValues(clusterName).Set(1.0)
+		return
+	}
+	binlogBackupInProgressMetrics.WithLabelValues(clusterName).Set(0.0)
+}
 
 func IncrementCloneCountMetrics(clusterName string) {
 	cloneCountMetrics.WithLabelValues(clusterName).Inc()
 }
-
 func IncrementCloneFailureCountMetrics(clusterName string) {
 	cloneFailureCountMetrics.WithLabelValues(clusterName).Inc()
 }
-
 func UpdateCloneDurationSecondsMetrics(clusterName string, durationSeconds float64) {
 	cloneDurationSecondsMetrics.WithLabelValues(clusterName).Observe(durationSeconds)
 }
+func SetCloneInProgressMetrics(clusterName string, inProgress bool) {
+	if inProgress {
+		cloneInProgressMetrics.WithLabelValues(clusterName).Set(1.0)
+		return
+	}
+	cloneInProgressMetrics.WithLabelValues(clusterName).Set(0.0)
+}
```

Comment:
Let's prepare `SetDumpInProgressMetrics()` too, though it will not be used for now.